lib/reek/smells/control_parameter.rb

```diff
@@ -192,6 +196,8 @@ def find_matches(param)
           ControlParameterFinder.new(context.exp, param).find_matches
         end
       end
+
+      private_constant :ControlParameterCollector
```

Comment:
Any thoughts on clustering these all at the end vs. putting the right after the class?